docs/source/api/v1/servers.rst

```diff
@@ -90,8 +90,16 @@ Response Structure
 :interfaceMtu:   The :abbr:`MTU (Maximum Transmission Unit)` configured on ``interfaceName``
 :interfaceName:  The name of the primary network interface used by the server
 :ip6Address:     The IPv6 address and subnet mask of ``interfaceName``
+:ip6IsService:   A boolean value which if ``true`` indicates that the IPv6 address will be used for routing content.
+
+	.. versionadded:: 2.0
```

Comment:
all the versionadded should be added to the v2.0 docs and removed completely from the v1 docs. Also when moving them no need to include versionadded anymore

Reply:
done